app/core/reservationbook.py

```diff
@@ -4,218 +4,215 @@
-        else:
-            rid = 0
-        return rid
-
-    # Method to generate waitingId
-    def genWid(self):
-        if len(self.waitingList) != 0:
-            largeList = []
-            for index in range(len(self.waitingList)):
-                largeList.append(self.waitingList[index].getId())
-            wid = max(largeList) + 1
-        else:
-            wid = 0
-        return wid
-
-    # Method to generate timeslotId
-    def genTid(self):
-        timeslotList = []
-        for index in range(len(self.waitingList)):
-            timeslotList.append(self.waitingList[index].getTimeslot())
-        for index in range(len(self.reservationList)):
-            timeslotList.append(self.reservationList[index].getTimeslot())
-
-        if len(timeslotList) != 0:
-            largeList = []
-            for index in range(len(timeslotList)):
-                largeList.append(timeslotList[index].getId())
-            tid = max(largeList) + 1
-        else:
-            tid = 0
-        return tid
-
-    # Method for restriction
-    def isRestricted(self,user,time):
-        restrictions = False
-        nbMyReservationInWeek = 0
-        # Get user reservations
-        myReservationList = self.viewMyReservation(user)
-
-        # Get week nb of specify Timeslot
-        date1 = time.getDate()
-        day1 = int(date1[8:10])
-        month1 = int(date1[5:7])
-        year1 = int(date1[0:4])
-        dt1 = datetime(year1, month1, day1)
-        wk1 = dt1.isocalendar()[1]
-
-        for index in range(len(myReservationList)):
-            r = myReservationList[index]
-            # Get week nb
-            date2 = r.getTimeslot().getDate()
-            day2 = int(date2[8:10])
-            month2 = int(date2[5:7])
-            year2 = int(date2[0:4])
-            dt2 = datetime(year2, month2, day2)
-            wk2 = dt2.isocalendar()[1]
-            # Compare if week nb matches
-            if wk1 == wk2:
-                nbMyReservationInWeek = nbMyReservationInWeek + 1
-            # Check if user is attempting to make another reservation on same day
-            if r.getTimeslot().getDate() == time.getDate():
-                 restrictions = True
-                 print("Request Failed: Only one reservation per day.")
-                 break
-        # Check if user is at max reservation
-        if nbMyReservationInWeek >= 3:
-            restrictions = True
-            print("Request Failed: At maximum number of reservations for this week.")
-
-        return restrictions
-
-    # Accessors and Mutators
-    def getReservationList(self):
-        return self.reservationList
-
-    def setReservationList(self, reservationList):
-        self.reservationList = reservationList
-
-    def getWaitingList(self):
-        return self.waitingList
-
-    def setWaitingList(self, waitingList):
-        self.waitingList = waitingList
+	# Default Constructor
+	def __init__(self):
+		self.reservationList = deque()
```

Comment:
I saw that they were there necessary only for some dummy tests, but since the tests have been deleted by Lance, I will delete these useless constructors.